src/ucs/memory/rcache.c

```diff
@@ -370,11 +369,24 @@ ucs_rcache_region_lru_put(ucs_rcache_t *rcache, ucs_rcache_region_t *region)
     ucs_spin_unlock(&rcache->lru.lock);
 }
 
+static size_t ucs_rcache_distribution_get_bin(size_t region_size)
+{
+    size_t i;
+
+    for (i = 0; i < ucs_rcache_distribution_get_num_bins(); ++i) {
+        if (region_size < UCS_RCACHE_STAT_MIN_POW2 << i) {
+            break;
+        }
+    }
```

Comment:
maybe use `ucs_count_leading_zero_bits` instead?

Reply:
Do you mean smth like this? 
```
size_t region_size_lz, threshold_lz, i;

region_size_lz = ucs_count_leading_zero_bits(region_size);
threshold_lz   = ucs_count_leading_zero_bits(UCS_RCACHE_STAT_MIN_POW2);

for (i = 0; i < ucs_rcache_distribution_get_num_bins() - 1; ++i) {
    if (region_size_lz > threshold_lz - i) {
        break;
    }
}

return i;
```